404.html

```diff
@@ -0,0 +1,13 @@
+<HTML>
```

Comment:
This files existence is questionable. It's not really needed here, but I am not going to ding you for it either.